src/Mjolnir.ts

```diff
@@ -493,8 +495,10 @@ export class Mjolnir {
 
         const aclErrors = await applyServerAcls(this.banLists, Object.keys(this.protectedRooms), this);
         const banErrors = await applyUserBans(this.banLists, Object.keys(this.protectedRooms), this);
+        const redactionErrors = await this.processRedactionQueue();
```

Comment:
It's not clear to me when this entire method is executed. Could you take the opportunity to document this?